scripts/pre_commit_linter.py

```diff
@@ -643,9 +643,10 @@ def _check_import_order(all_files):
     print 'Starting import-order checks'
     print '----------------------------------------'
     summary_messages = []
-    all_files = [
+    files_to_check = [
         filename for filename in all_files if not
-        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        and filename.endswith('.py')]
     failed = False
     for filename in all_files:
```

Comment:
change all_files to `files_to_check` where it's required.

Reply:
Done! Thanks, @DubeySandeep!